tests/page-objects/contacts/contacts.wdio.page.js

```diff
@@ -69,18 +79,54 @@ const addPlace = async (type, placeName , contactName ) => {
   await (await contactCard()).waitForDisplayed();
 };
 
-const addPerson = async (name, dob = '2000-01-01') => {
+const addPerson = async (name, params = {}) => {
+  const { dob='2000-01-01', phone } = params;
   await (await actionResourceIcon('person')).click();
   await (await personName()).addValue(name);
   await (await dateOfBirthField()).addValue(dob);
   await (await personName()).click(); // blur the datepicker field so the sex field is visible
+  if (phone) {
+    await (await personPhoneField()).addValue(phone);
+  }
   await (await personSexField()).click();
   await (await notes('person')).addValue('some person notes');
   await (await genericForm.submitButton()).click();
   await (await contactCardIcon('person')).waitForDisplayed();
   return (await contactCard()).getText();
 };
 
+const editPerson = async (name, updatedName) => {
+  await selectLHSRowByText(name);
+  await waitForContactLoaded();
+  await (await editContactButton()).click();
+
+  await (await genericForm.nextPage());
+
+  await (await personName()).clearValue();
+  await (await personName()).addValue(updatedName);
+
+  await (await genericForm.submitButton()).click();
+  await waitForContactLoaded();
+  return (await contactCard()).getText();
+};
+
+const waitForContactLoaded = async () => {
+  await (await contactCard()).waitForDisplayed();
+  await (await contactSummaryContainer()).waitForDisplayed();
+};
+
+const getContactSummaryField = async (fieldName) => {
+  await (await contactSummaryContainer()).waitForDisplayed();
+  const fields = await (await contactSummaryContainer()).$$('.cell');
+  for (const field of fields) {
+    const classNames = (await field.getAttribute('class')).split(' ');
+    if (classNames.includes(fieldName)) {
+      const fieldValue = await field.$('p');
```

Comment:
It's too bad this element doesn't have any class. I'm afraid tests might break in the future because of this selector

Reply:
It has a class, but the class contains the dot character (`.`) which makes selecting it via class name impossible.

Reply:
Have you tried using two backslashes `$(".person\\.field\\.phone")` ?

Reply:
I was thinking about the ongoing enketo upgrade that could use different markup and breaking selectors with low specificity like `$('p')` here but I might be overthinking it

Reply:
The enketo upgrade won't change how we display contact summary fields.